### PR TITLE
KOGITO-8072: Flaky test - extension-editors-bpmn.test.ts

### PR DESCRIPTION
**Pull request by zdrapela**
**Jira:** https://issues.redhat.com/browse/KOGITO-8072

### CODE REVIEW

**Comment by zdrapela** [comment]
Hi, sorry for the mess with the force-push. I needed to force a partial build (instead of full) on my fork to verify that multiple runs directly run on the CI won't fail. I should have used a different branch. 
So far, with the partial build, I got 5 passes with [ubuntu-latest](https://github.com/zdrapela/kie-tools/actions/runs/3280202519/jobs/5402535553) and no fail there.

File: packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors-bpmn.test.ts [hunk 1]
```diff
@@ -243,7 +244,7 @@ describe.skip("KIE Editors Integration Test Suite - BPMN Editor", () => {
   });
 
   it("Opens UserTask.bpmn file in BPMN Editor and test On Entry and On Exit actions", async function () {
```
Comment (domhanak) [inline packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors-bpmn.test.ts:246]: so can we split this into two smallers units?

Comment (zdrapela) [reply packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors-bpmn.test.ts:246]: It is a really small test and there is not much to be split.

**Comment by zdrapela** [comment]
Hi, what do you think about this PR @yesamer @tiagobento ?
The test is stable on my machine (even when low compute power is dedicated to running the test), and also multiple runs of a partial build of the affected package on CI runs without fail. ~~I see there are a few failed checks, but they seem unrelated and I believe they will go green when rerun.~~ All the checks are now green.

**Review by yesamer — APPROVED** [review]
@zdrapela Thank you. I agree with your assumption, in several cases, I saw Timeout exceptions during the testing phase, so I suppose the CI/CD performs these tests slower than in our local machines. However, this couldn't be enough to stabilize the testing phase at all, so I kindly ask you to monitor the testing phase situation in the near future. Thank you.

**Comment by zdrapela** [comment]
Sure, I will keep an eye on it. Thank you @yesamer

**Review by tiagobento — APPROVED** [review]

**Pull request merged**